.codegen/_openapi_sha

```diff
@@ -1 +1 @@
-fbdd0fa3e83fed2c798a58d376529bdb1285b915
\ No newline at end of file
+universe:/Users/miles/universe
```

Comment:
looks very unintentional ;) i'd prefer if we remove the ability to put rfs into this file.

PS: I just associate `refish` with "a fish named RE". Can we change that? =P

Reply:
I suppose we could call it `ref`?

Reply:
ref is better :)